.github/workflows/release.yml

```diff
@@ -37,20 +37,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        arch: [auto, aarch64, universal2]
+        os: [ubuntu-latest, windows-latest, macos-13, macos-14]
```

Comment:
MacOS 14 is the new ARM runner. So this builds native wheels on both (the ARM part can now be tested! Except for CPython 3.8, which is wonky)